CRUDEFutures/CRUDE.swift

```diff
@@ -215,28 +214,7 @@ public struct CRUDE {
         return NSError(domain: issue, code: (network.response?.statusCode ?? -1), userInfo: debugInfo)
     }
 
-    private static var defaultLogger: CRUDELog = { type, network in
```

Comment:
The default logger was a requested feature I added. While I will agree that different developer teams will have different logging preferences, an option to use a pre-baked one does no harm.